client/src/components/CreateForm/CreateForm.js

```diff
@@ -126,7 +126,7 @@ const CreateForm = ({ formType, onSuccess, onError, onClose, tripId }) => {
   const currentUser = useSelector((state) => state.get('auth').user).username;
 
   useEffect(() => {
-    axios.get('http://localhost:3001/user').then(
+    axios.get('/user').then(
```

Comment:
where did you set the axios base URL? is there a conditional logic such as 
```
axios.defaults.baseURL =
  process.env.NODE_ENV === 'development'
    ? process.env.REACT_APP_BE_URL
    : 'https://xyz.herokuapp.com';
```

Reply:
I didn't have to do this for my assignment. Axios seems to manage this by itself.

Reply:
yep. axios manages this by itself